src/tag-attr-list.js

```diff
@@ -121,82 +119,82 @@ Whitelist.Tags = [
     "wbr"
 ];
 
-Whitelist.Attributes = [
+exports.WhiteListAttributes = [
     "ept",
-    "acceptCharset",
-    "accessKey",
+    "acceptcharset",
+    "accesskey",
     "action",
-    "allowFullScreen",
-    "allowTransparency",
+    "allowfullscreen",
+    "allowtransparency",
     "alt",
     "async",
-    "autoComplete",
-    "autoFocus",
-    "autoPlay",
-    "cellPadding",
-    "cellSpacing",
-    "charSet",
+    "autocomplete",
+    // "autofocus",
+    "autoplay",
+    "cellpadding",
+    "cellspacing",
+    // "charset",
```

Comment:
>  Allowing <iframe src="http://attacker.com"> is not ideal too.

We can handle this better by allowing user to specify a whitelist of domains. For eg: we should allow ```<iframe src="http://mydomain.com">``` but filter the value for ```<iframe src="http://attacker.com">```